### PR TITLE
Add steps to verify VS Code Terminal + Hyper setup

### PR DESCRIPTION
**Pull request by ProchaLu**
## Description

Added new steps to the system setup guide to help students verify a successful VS Code and Hyper setup. Previously, there were no instructions for checking whether the VS Code and Hyper changes were applied correctly, and students often encountered issues with PowerShell in the terminal when starting the Bootcamp. This should help students avoid any initial setup issues and ensure a smoother start to the Bootcamp.

### CODE REVIEW

**Comment by karlhorky** [comment]
Hmm... the thing in the screenshot does not have to do with Hyper - it is VS Code

There should be a separate verification step + 2x screenshots for Hyper - can be same PR

File: windows.md [hunk 1]
```diff
@@ -193,8 +193,22 @@ With those compatibility things out of the way, you're ready to start the system
...
+    ```
+
+    You should see the following output:
+   
+    ```bash
+    C:\Program Files\Git\bin\bash.exe
+    cygwin
+    ```
+    
+    If you see this output, then the Hyper setup was successful. If the output is different, then you need to go back to step 1 and double-check your changes
```
Comment (karlhorky) [inline windows.md:211]: going back to step 1 seems like a long way back 😬  (this is step 16 already) is it possible to do a smaller step back?

Comment (ProchaLu) [reply windows.md]: Changed it, now it says go back to step 16
[d299d98](https://github.com/upleveled/system-setup/pull/32/commits/d299d98bc3aba9bd5b895910fb57503e73f8d8a1)

File: windows.md [hunk 2]
```diff
@@ -193,8 +193,22 @@ With those compatibility things out of the way, you're ready to start the system
 
     1. Find `shell: '',` and replace it with `shell: 'C:\\Program Files\\Git\\bin\\bash.exe',`
     2. Find `env: {},` and replace it with `env: { TERM: 'cygwin' },`
+    3. Save the file and close and restart Hyper
 
-    Then save the file and close and restart Hyper.
+    To test that the Hyper setup was successful, you can copy and paste the following command into Hyper and hit return:
+    
+    ```bash
+    echo $SHELL; echo $TERM
```
[the remark this answers is not in the document]
Comment (ProchaLu) [reply windows.md]: Added the VS Code check 
[d299d98](https://github.com/upleveled/system-setup/pull/32/commits/d299d98bc3aba9bd5b895910fb57503e73f8d8a1)

File: windows.md [hunk 3]
```diff
@@ -193,8 +193,28 @@ With those compatibility things out of the way, you're ready to start the system
 
     1. Find `shell: '',` and replace it with `shell: 'C:\\Program Files\\Git\\bin\\bash.exe',`
     2. Find `env: {},` and replace it with `env: { TERM: 'cygwin' },`
-
-    Then save the file and close and restart Hyper.
+    3. Save the file and close and restart Hyper.
+    4. To verify that the setup was successful, you can copy and paste the following command into Hyper and hit return:
+    
+        ```bash
+        echo $SHELL; echo $TERM
```
Comment (karlhorky) [inline windows.md:200]: let's avoid semicolons - we don't use these in our commands

File: windows.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (karlhorky) [reply windows.md]: ah I meant after the VS code settings setup

[the remark this answers is not in the document]
Comment (ProchaLu) [reply windows.md]: Added the step to the VS Code settings setup
[129e30b](https://github.com/upleveled/system-setup/pull/32/commits/129e30b1b6944889da8ae65f20da318282e99710)

File: windows.md (hunk 3 above)
Comment (ProchaLu) [reply windows.md]: Changed the command to a single echo and added a prefix
[129e30b](https://github.com/upleveled/system-setup/pull/32/commits/129e30b1b6944889da8ae65f20da318282e99710)

File: windows-16-vscode-hyper.jpg
Comment (karlhorky) [inline windows-16-vscode-hyper.jpg:1]: The screenshots can be removed again?

File: windows.md [hunk 4]
```diff
@@ -186,15 +186,43 @@ With those compatibility things out of the way, you're ready to start the system
...
+    2. Copy and paste the following command into the terminal and hit return:
+      
+       ```bash
+       echo "[VS Code config] Shell: $SHELL | Terminal: $TERM"
+       ```
+     
+       This should display:
+     
+       ```bash
+       [VS Code config] Shell: /usr/bin/bash | Terminal: xterm-256color
```
Comment (karlhorky) [inline windows.md:200]:
```suggestion
       [VS Code config]
       Shell: /usr/bin/bash
       Terminal: xterm-256color
```

File: windows.md [hunk 5]
```diff
@@ -186,15 +186,45 @@ With those compatibility things out of the way, you're ready to start the system
...
 
     1. Find `shell: '',` and replace it with `shell: 'C:\\Program Files\\Git\\bin\\bash.exe',`
     2. Find `env: {},` and replace it with `env: { TERM: 'cygwin' },`
-
-    Then save the file and close and restart Hyper.
+    3. Save the file and close and restart Hyper
+    4. To verify that the setup was successful, copy and paste the following command into Hyper and hit return:
+    
+        ```bash
+        echo "[Hyper config] Shell: $SHELL | Terminal: $TERM"
```
Comment (karlhorky) [inline windows.md:218]:
```suggestion
        echo "[Hyper config]\nShell: $SHELL\nTerminal: $TERM"
```

File: windows.md [hunk 6]
```diff
@@ -186,15 +186,45 @@ With those compatibility things out of the way, you're ready to start the system
     If you have any of these warnings, we should fix them. For each one of these lines with the warnings on them, delete the full line, including the comma at the end. We usually like to select from the start of the first `"` to just before the next `"` on the next line:<br><br>
     <img src="./general-vscode-settings-fix-warnings.png"><br><br>
 
+    To verify that the setup for Git Bash was successful, we will open a new terminal window in VS Code:
+    1. Click on `Terminal` in the top menu and select `New Terminal`
+    2. Copy and paste the following command into the terminal and hit return:
+      
+       ```bash
+       echo "[VS Code config]\nShell: $SHELL\nTerminal: $TERM"
```
Comment (karlhorky) [inline windows.md:194]:
```suggestion
       echo -e "[VS Code config]\nShell: $SHELL\nTerminal: $TERM"
```

**Pull request merged**